app.js

```diff
@@ -39,7 +39,7 @@ const discord = new Discord.Client({
     Discord.GatewayIntentBits.MessageContent,
   ]
 })
-discord.login(process.env.DISCORD_BOT_TOKEN)
+// discord.login(process.env.DISCORD_BOT_TOKEN)
```

Comment:
Probably this was commented out for testing? I'm a git noob and am not sure if there's an easy way for me to just nix this change as part of merging the PR?

Reply:
fixed in https://github.com/dreeves/lexiguess/pull/65/commits/874bf5df813e1d8e8ddbec06131726fea6b12e4f